tokenizer.js

```diff
@@ -197,7 +199,7 @@
 				limit = options.numToSuggest || 1000,
 				list = [];
 				for (i = 0; word && i < words.length && list.length < limit; i++) {
-					if (!words[i].match(re1)) { continue; }
+					if (!words[i].match(re1) && !(options.allowFullSearch && word == options.fullSearchChar)) { continue; }
```

Comment:
The idea was different. I'd like to have 'select like behavior'. When you open a combobox you can see all predefined options. You cannot type any new tag, you can just select one from the list. However when your list is long (e.g. 20-50 options) than you have to remember all options or put at least one letter and check list over and over. 

My idea was to use * to show all suggestions without limitation (that's why I changed 5 to 0). 

At the end I didn't want to show list of suggestion on focus :) With minChar options there could be a little problem. What about the situation when you click on input by accident. You'll see the big list of options. On the other side when you enter at least one letter than you have filtered list. 

To tackle this problem it I used * to show all suggestions or any other letter to keep standard filter behavior.